soc/ite/ec/it51xxx/linker.ld

```diff
@@ -443,6 +453,17 @@ GROUP_END(DTCM)
 
     GROUP_END(RAMABLE_REGION)
 
+#ifdef CONFIG_SOC_IT51XXX_SHA256_HW_ACCELERATE
+	GROUP_START(SHA256_DLM)
+
+		SECTION_PROLOGUE(it515xx_sha256_ram, (NOLOAD),)
+		{
```

Comment:
Since sha256 dlm offset can be reconfigured, add build assertion to ensure this section is 4k aligned.